test/kitchen/site-cookbooks/dd-security-agent-check/recipes/stress-tests.rb

```diff
@@ -5,22 +5,14 @@
 # Copyright (C) 2020-present Datadog
 #
 
-if node['platform_family'] != 'windows'
-  wrk_dir = '/tmp/security-agent'
+cookbook_file "#{node['common']['work_dir']}/tests/stresssuite" do
+  source "tests/stresssuite"
+  mode '755'
+end
```

Comment:
Related to the above, when running the `security-agent-stress` test suite, `default.rb` isn't run, so 
```
directory "#{node['common']['work_dir']}/tests" do
  recursive true
end
```
isn't part of the `run_list`. That may be an issue if the directory isn't created when this `cookbook_file` command is run?

Reply:
Commented in the other comment, but updated the run list!